MapData.py

```diff
@@ -1,41 +1,31 @@
-import os
 import threading
-from enum import Enum
 
-import matplotlib.pyplot as plt
-from scipy.misc import imresize
 
-from detail import LOCAL
+ZOOM = 'zoom'
+RADIUS = 'radius'
+IMAGE = 'image'
+MARK = 'mark'
+LOCATION = 'location'
```

Comment:
```suggestion
ZOOM = '_zoom'
RADIUS = '_radius'
IMAGE = '_image'
MARK = '_mark'
LOCATION = '_location'
```
Maybe just to signify that these are private. I dont think ppls IDEs would even pick up on them tho.